apps/synt-sykemelding-api/src/main/java/no/nav/registre/testnorge/synt/sykemelding/consumer/dto/SyntDiagnoserDTO.java

```diff
@@ -0,0 +1,16 @@
+package no.nav.registre.testnorge.synt.sykemelding.consumer.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import lombok.Value;
+
+@Value
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(force = true)
+public class SyntDiagnoserDTO {
+    String dn;
+    String s;
+    String v;
```

Comment:
Greier vi å ha noen mer beskrivende variabelnavn her og bruke JsonProperty for å mappe? (Gitt at vi greier å finne ut hva disse forkortelsene betyr xD)

Reply:
Problmet var at jeg heller ikke vet 😅

Reply:
Fant ut av det 😄

Reply:
Niiice! :D